src/main/java/de/tdlabs/examples/keycloak/EmbeddedKeycloakApp.java

```diff
@@ -8,25 +11,37 @@
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.ApplicationListener;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.PropertySource;
 
 @SpringBootApplication(exclude = LiquibaseAutoConfiguration.class)
 @EnableConfigurationProperties(KeycloakServerProperties.class)
-public class EmbeddedKeycloakApp {
+@PropertySource(value = "classpath:keycloak.properties")
+@PropertySource(value = "classpath:keycloak-user.properties", ignoreResourceNotFound = true)
+public class EmbeddedKeycloakApp extends StaticPropertyUtil {
+
+  private static final Logger LOG   = LogManager.getLogger(EmbeddedKeycloakApp.class);
+
+  protected static KeycloakServerProperties.AdminUser getAdminUser() {
+    return StaticPropertyUtil.getAdminUser();
+  }
+
+  protected static String getContextPath(){
+    return StaticPropertyUtil.getContextPath();
+  }
```

Comment:
Is it really necessary to extend StaticPropertyUtil. Can't we just refer to StaticPropertyUtil whenever we need it (e.g. in the onApplicationReadyEventListener)?